README.md

```diff
@@ -29,42 +29,50 @@ Implement an User Authentication System in order to access the jokes from the Jo
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. What is the purpose of using _sessions_?
+   -the purpose of using sessions is to persist data.
 1. What does bcrypt do to help us store passwords in a secure manner.
-1. What does bcrypt do to slow down attackers?
+
+- bcrypt help us randomize the password by hashing it so no real passwords are stored in the database.
+
+1. What does bcrypt do to slow down attackers.
+
+- it hashes the password with a key derivation function. It also adds a random string(salt) to increase password length.
```

Comment:
Good job looking up the `key derivation function`, second person I've seen do this, shows you're looking stuff on your own.